context/build.gradle.kts

```diff
@@ -70,7 +70,7 @@ testing {
 }
 
 tasks {
-  named("check") {
```

Comment:
I think this should be kept as-is to profit from conviguration-avoidance: https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:old_vs_new_configuration_api_overview

Reply:
It's not clear from the docs but it seems the kotlin dsl for these seems to use the lazy configuration APIs based on code navigating. Switched for the compile time safety